utils/build_swift/driver_arguments.py

```diff
@@ -572,6 +582,11 @@ def create_argument_parser():
         help="run the validation test suite (implies --test)",
         action=arguments.action.optional_bool)
     run_tests_group.add_argument(
+        "--test-paths",
+        help="run tests located in specific directories and/or files",
```

Comment:
Since this new `--test-paths` option *can* imply `--test` and/or `--validation-test` would you mind adding that to the help text?

Reply:
Sure, will do!